Added matrixTranspose.java to misc folder

Rajat-Jain29 opened the pull request:
### **Describe your change:**

- [x] Add an algorithm?
- [x] Fix a bug or typo in an existing algorithm?
- [x] Documentation change?

#### References

<!-- Add any reference to previous pull-request or issue -->

### **Checklist:**

<!-- Remove items that do not apply. For completed items, change [ ] to [x]. -->

- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Java/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized.
- [x] I know that pull requests will not be merged if they fail the automated tests.
- [x] This PR only changes one algorithm file. To ease review, please open separate PRs for separate algorithms.
- [x] All new Java files are placed inside an existing directory.
- [x] All filenames are in all uppercase characters with no spaces or dashes.
- [x] All functions and variable names follow Java naming conventions.
- [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
- [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

ayaankhan98 commented on Misc/matrixTranspose.java:
```diff
@@ -0,0 +1,39 @@
+package math;
+
+import java.util.*;
```
import only required classes instead of import everything

ayaankhan98 reviewed (commented):
Thanks for your contribution!
please write relevant documentaion and remove the `.yaml.txt` file from the PR

Rajat-Jain29 commented:
Sir But problem is that check Code formator is not clear so please provide
some solution ao that I can clear

On Sun, 22 Nov, 2020, 10:57 pm Ayaan Khan, <notifications@github.com> wrote:

> *@ayaankhan98* commented on this pull request.
> ------------------------------
>
> In Misc/matrixTranspose.java
> <https://github.com/TheAlgorithms/Java/pull/2041#discussion_r528366142>:
>
> > @@ -0,0 +1,39 @@
> +package math;
> +
> +import java.util.*;
>
> import only required classes instead of import everything
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/TheAlgorithms/Java/pull/2041#pullrequestreview-536083596>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AR2RIZRYI2BHDT3QNNCRGT3SRFCYXANCNFSM4T56CUKA>
> .
>

ayaankhan98 commented:
> Sir But problem is that check Code formator is not clear so please provide some solution ao that I can clear
> […](#)
> On Sun, 22 Nov, 2020, 10:57 pm Ayaan Khan, ***@***.***> wrote: ***@***.**** commented on this pull request. ------------------------------ In Misc/matrixTranspose.java <[#2041 (comment)](https://github.com/TheAlgorithms/Java/pull/2041#discussion_r528366142)>: > @@ -0,0 +1,39 @@ +package math; + +import java.util.*; import only required classes instead of import everything — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub <[#2041 (review)](https://github.com/TheAlgorithms/Java/pull/2041#pullrequestreview-536083596)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AR2RIZRYI2BHDT3QNNCRGT3SRFCYXANCNFSM4T56CUKA> .

@Rajat-Jain29 No need to worry about that code formatter seems to be broken. that's not your error

Rajat-Jain29 commented:
Sir I removed that file also changed the code but still it doesn't solve .
Please help me sir

On Sun, 22 Nov, 2020, 11:17 pm Ayaan Khan, <notifications@github.com> wrote:

> Sir But problem is that check Code formator is not clear so please provide
> some solution ao that I can clear
> … <#m_-1963807845220539789_>
> On Sun, 22 Nov, 2020, 10:57 pm Ayaan Khan, *@*.*> wrote: @.** commented
> on this pull request. ------------------------------ In
> Misc/matrixTranspose.java <#2041 (comment)
> <https://github.com/TheAlgorithms/Java/pull/2041#discussion_r528366142>>:
> > @@ -0,0 +1,39 @@ +package math; + +import java.util.*; import only
> required classes instead of import everything — You are receiving this
> because you authored the thread. Reply to this email directly, view it on
> GitHub <#2041 (review)
> <https://github.com/TheAlgorithms/Java/pull/2041#pullrequestreview-536083596>>,
> or unsubscribe
> https://github.com/notifications/unsubscribe-auth/AR2RIZRYI2BHDT3QNNCRGT3SRFCYXANCNFSM4T56CUKA
> .
>
> @Rajat-Jain29 <https://github.com/Rajat-Jain29> No need to worry about
> that code formatter seems to be broken. that's not your error
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/TheAlgorithms/Java/pull/2041#issuecomment-731783556>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AR2RIZXL7ZPSPYWWETZIVATSRFFBLANCNFSM4T56CUKA>
> .
>

ayaankhan98 commented:
please write relevant documentaion

Rajat-Jain29 commented:
Sir can you please explain what is relevant documentation .??
Because sir its my first time of contribution so I don't have any
idea about it
So please give some idea about documentation please sir

On Sun, 22 Nov, 2020, 11:43 pm Ayaan Khan, <notifications@github.com> wrote:

> please write relevant documentaion
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/TheAlgorithms/Java/pull/2041#issuecomment-731799552>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AR2RIZRZLKCKMVHP5Y4OIFTSRFIEFANCNFSM4T56CUKA>
> .
>

ayaankhan98 commented:
> Sir can you please explain what is relevant documentation .?? Because sir its my first time of contribution so I don't have any idea about it So please give some idea about documentation please sir
> […](#)
> On Sun, 22 Nov, 2020, 11:43 pm Ayaan Khan, ***@***.***> wrote: please write relevant documentaion — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub <[#2041 (comment)](https://github.com/TheAlgorithms/Java/pull/2041#issuecomment-731799552)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AR2RIZRZLKCKMVHP5Y4OIFTSRFIEFANCNFSM4T56CUKA> .

Documentation basically refers to comments which explains your code properly.
please refer [here](https://www.tutorialspoint.com/java/java_documentation.htm#:~:text=Javadoc%20is%20a%20tool%20which,documentation%20in%20a%20predefined%20format.)

Rajat-Jain29 commented:
Basically you are saying that I add Comment to my code ??
Aur Make another html file in that file write raw code with comment please
clarity it sir

On Sun, 22 Nov, 2020, 11:51 pm Ayaan Khan, <notifications@github.com> wrote:

> Sir can you please explain what is relevant documentation .?? Because sir
> its my first time of contribution so I don't have any idea about it So
> please give some idea about documentation please sir
> … <#m_8825113221782224881_m_2007070244731273886_>
> On Sun, 22 Nov, 2020, 11:43 pm Ayaan Khan, *@*.***> wrote: please write
> relevant documentaion — You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <#2041 (comment)
> <https://github.com/TheAlgorithms/Java/pull/2041#issuecomment-731799552>>,
> or unsubscribe
> https://github.com/notifications/unsubscribe-auth/AR2RIZRZLKCKMVHP5Y4OIFTSRFIEFANCNFSM4T56CUKA
> .
>
> Documentation basically refers to comments which explains your code
> properly.
> please refer here
> <https://www.tutorialspoint.com/java/java_documentation.htm#:~:text=Javadoc%20is%20a%20tool%20which,documentation%20in%20a%20predefined%20format.>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/TheAlgorithms/Java/pull/2041#issuecomment-731804151>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AR2RIZTKXMZ4U3K7IWJZBVLSRFJBTANCNFSM4T56CUKA>
> .
>

Rajat-Jain29 commented:
Sir I uploaded code with so many comments but still my checks are not clear

On Mon, 23 Nov, 2020, 12:07 am Rajat Jain, <rajatjain.jain29@gmail.com>
wrote:

> Basically you are saying that I add Comment to my code ??
> Aur Make another html file in that file write raw code with comment please
> clarity it sir
>
> On Sun, 22 Nov, 2020, 11:51 pm Ayaan Khan, <notifications@github.com>
> wrote:
>
>> Sir can you please explain what is relevant documentation .?? Because sir
>> its my first time of contribution so I don't have any idea about it So
>> please give some idea about documentation please sir
>> … <#m_-3584556919658169781_m_8825113221782224881_m_2007070244731273886_>
>> On Sun, 22 Nov, 2020, 11:43 pm Ayaan Khan, *@*.***> wrote: please write
>> relevant documentaion — You are receiving this because you were mentioned.
>> Reply to this email directly, view it on GitHub <#2041 (comment)
>> <https://github.com/TheAlgorithms/Java/pull/2041#issuecomment-731799552>>,
>> or unsubscribe
>> https://github.com/notifications/unsubscribe-auth/AR2RIZRZLKCKMVHP5Y4OIFTSRFIEFANCNFSM4T56CUKA
>> .
>>
>> Documentation basically refers to comments which explains your code
>> properly.
>> please refer here
>> <https://www.tutorialspoint.com/java/java_documentation.htm#:~:text=Javadoc%20is%20a%20tool%20which,documentation%20in%20a%20predefined%20format.>
>>
>> —
>> You are receiving this because you were mentioned.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/TheAlgorithms/Java/pull/2041#issuecomment-731804151>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AR2RIZTKXMZ4U3K7IWJZBVLSRFJBTANCNFSM4T56CUKA>
>> .
>>
>

siriak reviewed (approved):
Thanks!
